python/Makefile

```diff
@@ -0,0 +1,12 @@
+PYTHON_BIN := python
```

Comment:
Please don't mix C++ and Python build environment (like `Makefile`s) in opensource world.